Unicode/makeutype.c

```diff
@@ -177,15 +187,53 @@ static void FigureAlternates(long index, char *apt, int normative) {
 	alts[alts[index][0]][0] = index;
 }
 
-static void processAssignment(long index,char *pt) {
+static int processAssignment(long index,char *pt,long *flg) {
     static long first=-1;
     long i;
 
-    if ( index>0x11ffff )
-return;
+    if ( index<0 ) return( -1 );
+    if ( index>0x11ffff ) return( 0 );
     ++pt;							/* move past semicolon */
```

Comment:
This is not whitespace, and you're already adjusting this file. Surely adding `if (pt[0] == '\0') return -1` would not ruin your day? I'm happy to do it if necessary, but it's much more efficient if you do it.